### ISSUE TITLE
JSON format stop working after adding `"source.organizeImports.biome": true`

### ISSUE THREAD

**Issue by chungweileong94** [biomejs/biome-zed#26]
The JSON format stop working after enabling the organize import options:
```json
"code_actions_on_format": {
     "source.organizeImports.biome": true
}
```

https://github.com/biomejs/biome-zed/assets/15154097/ff5b5630-3632-46c7-8da6-8610d364883e

---

### System Spec
Zed: v0.142.3 (Zed Preview)
OS: macOS 14.5.0
Memory: 16 GiB
Architecture: aarch64
Biome: 1.8.3

**Comment by chungweileong94** [comment]
Haven't spend too much time on debugging the issue, but it could be the code action is not working correctly in the latest Zed preview.

**Comment by beckend** [comment]
Workaround:

```json
{
  "languages": {
    "JSON": {
      "code_actions_on_format": {
        "source.organizeImports.biome": false
      }
    },
    "JSONC": {
      "code_actions_on_format": {
        "source.organizeImports.biome": false
      }
    }
  }
}
```

**Comment by chungweileong94** [comment]
> Workaround:
> 
> ```json
> {
>   "languages": {
>     "JSON": {
>       "code_actions_on_format": {
>         "source.organizeImports.biome": false
>       }
>     },
>     "JSONC": {
>       "code_actions_on_format": {
>         "source.organizeImports.biome": false
>       }
>     }
>   }
> }
> ```

Ah good idea!

**Comment by chungweileong94** [comment]
The problem seems to be fixed in Zed 0.146+.

**Comment by klaemo** [comment]
I still can't get it to work with zed `0.146.3` and biome `1.8.3`.

```json
{
  "formatter": {
    "language_server": {
      "name": "biome"
    }
  },
  "code_actions_on_format": {
    "source.organizeImports.biome": true
  }
}
```

`biome check . --write` works just fine and sorts the imports. Also, all other formatting (on save and on demand) gets applied as well.
